qa/automation/CareerPortal.Tests.Scenarios/4. Sending CV/4.1.Sending CV.feature

```diff
@@ -0,0 +1,107 @@
+Feature: 4.1. Sending CV
+
+@Type:Positive
+@Priority:Critical
+Scenario Outline: 4.1_01 Correct display of the cv submit form
```

Comment:
Let's just say "Send CV" form

Reply:
Thanks for your comment!
Ok, I'll fix it.